vdb/__init__.py

```diff
@@ -1321,14 +1321,20 @@ def do_gui(self, line):
             return
 
         import vqt.main as vq_main
-        import vdb.qt.main as vdb_q_main
-        import vqt.colors as vq_colors
+        if vq_main.isGuiStarted():
```

Comment:
Is this just the case for when somebody calls do_gui twice, or are we accidentally somewhere ourselves?

Reply:
this allows us to remove the "I MUST BE THE GUI" problem from vdbbin... allowing vdbbin to start the QApplication if it's not already, or *not* if it is.  you know how finicky the gui thread can be.